test/system/src/test/java/io/pravega/test/system/AbstractReadWriteTest.java

```diff
@@ -552,6 +573,89 @@ private EventWriterConfig buildWriterConfig() {
         return validEvents;
     }
 
+    void createScope(String scopeName, StreamManager streamManager) {
```

Comment:
I'm sure that there is a `createScope` method that you can reuse instead of creating a new one.

Reply:
Resolved, used existing createScope method